CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE Debug)
 endif()
 
+if (EMSCRIPTEN)
+    set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/os/emscripten/cmake")
+endif()
```

Comment:
Could consider generalising this to all OSes, and just do a check if the folder exists

Reply:
If we get more than one, yes. But given we only have one, I rather not premature generalise :)